theatre/studio/src/panels/SequenceEditorPanel/layout/tree.ts

```diff
@@ -22,6 +22,9 @@ export type SequenceEditorTree_Row<Type> = {
   depth: number
   top: number
   n: number
+  trackId?: string
+  sheetObject?: SheetObject
```

Comment:
These are available in `SequenceEditorTree_PrimitiveProp`, which I think is the only place we'd need them right now